edumanage/views.py

```diff
@@ -1862,28 +1862,31 @@ def closest(request):
         closest = -1
         points = getPoints()
         for (counter, i) in enumerate(points):
-            pointname = i['text']
-            pointlng = i['lng']
-            pointlat = i['lat']
-            pointtext = i['text']
-            plainname = i['name']
-            dLat = rad(float(pointlat) - float(lat))
-            dLong = rad(float(pointlng) - float(lng))
-            a = math.sin(dLat / 2) * math.sin(dLat / 2) + math.cos(rad(lat)) *\
-                math.cos(rad(float(pointlat))) * math.sin(dLong / 2) *\
-                math.sin(dLong / 2)
-            c = 2 * math.atan2(math.sqrt(a), math.sqrt(1 - a))
-            d = R * c
-            distances[counter] = d
-            if (closest == -1 or d < distances[closest]):
-                closest = counter
-                closestMarker = {
-                    'name': pointname,
-                    'lat': pointlat,
-                    'lng': pointlng,
-                    'text': pointtext,
-                    'plainname': plainname
-                }
+            try:
+                pointname = i['text']
+                pointlng = i['lng']
+                pointlat = i['lat']
+                pointtext = i['text']
+                plainname = i['name']
+                dLat = rad(float(pointlat) - float(lat))
+                dLong = rad(float(pointlng) - float(lng))
```

Comment:
The exception catching block should be limited to float casting statements.

Reply:
Hi Zenon, good catch - just adjusted the PR to do this.